strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/ActuatorEndpointControllerTest.java

```diff
@@ -110,7 +110,6 @@ public void testEndpointsWithUnauthorizedUser(final String endpoint)
     @ParameterizedTest
     @ValueSource(strings = { "/",
                              "/health",
-                             "/health/db",
```

Comment:
Are there any end-points we can use for this for Janusgraph, or Cassandra?

cc: @steve-todorov

Reply:
I haven't researched this

Reply:
Okay, in this case. Please, create a follow-up task for it, so we can research it. Thanks! :)

Reply:
There are for `Cassandra` which as far as I remember works only with Spring Data. There is none for JanusGraph for sure.